src/global.ts

```diff
@@ -1,7 +1,7 @@
-import * as cp from "cross-spawn";
-import * as fs from "fs-extra";
-import * as glob from "glob";
-import * as path from "path";
+import * as cp from "cross-spawn"
+import * as fs from "fs-extra"
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Missing semicolon ([external_tslint:semicolon](https://sonarcloud.io/organizations/1c-syntax/rules#rule_key=external_tslint%3Asemicolon))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=vsc-language-bsl-plugin&pullRequest=99&issues=AWYw2deTvOWp7Yd6ZJKY&open=AWYw2deTvOWp7Yd6ZJKY)

[ISSUE_KEY]: AWYw2deTvOWp7Yd6ZJKY